src/changelog/unreleased.md

```diff
@@ -134,6 +134,8 @@ changelog entry.
   - Rename `CursorEntered` to `PointerEntered`.
   - Rename `CursorLeft` to `PointerLeft`.
   - Rename `MouseInput` to `PointerButton`.
+  - Add `primary` to every `PointerEvent` as a way to identify discard non-primary pointers in a
+    multi-touch interaction.
   - Add `position` to every `PointerEvent`.
```

Comment:
Any specific reason why these are duplicated between `### Changed` here and `### Added` above?

Reply:
Yeah, its because we didn't really add a new event per se, but we changed the name of an existing event.
But the pointer event overhaul is a bit too big to describe with just "what changed", so the idea was to add what changed on a syntactic level for easy migration in the `### Changed` section and add an actual description of the new API in the `### Added` one.

I'm happy to take any suggestions!